core/src/data.ts

```diff
@@ -210,6 +212,79 @@ export class DrumRollConverter extends DrumsConverter {
   }
 }
 
+/**
+ * Converts between a quantized `NoteSequence` containing a drum sequence
+ * and the `Tensor` objects used by `MusicRNN`.
+ *
+ * The `Tensor` output by `toTensor` is a 2D one-hot encoding. Each
+ * row is a time step, and each column is a one-hot vector where each drum type
+ * is mapped to a single bit of a binary integer representation, where the
+ * bit has value 0 if the drum type is not present, and 1 if it is present.
+ *
+ * The expected `Tensor` in `toNoteSequence` is the same kind of one-hot
+ * encoding as the `Tensor` output by `toTensor`.
+ *
+ * The output `NoteSequence` uses quantized time and only the first pitch in
+ * pitch class are used.
+ *
+ * @param numSteps The length of each sequence.
+ * @param numSegments (Optional) The number of conductor segments, if
+ * applicable.
+ * @param pitchClasses (Optional) An array of arrays, grouping together MIDI
+ * pitches to treat as the same drum. The first pitch in each class will be used
+ * in the `NoteSequence` returned by `toNoteSequence`. A default mapping to 9
+ * classes is used if not provided.
+ */
+export class DrumsOneHotConverter extends DrumsConverter {
+
+  constructor(args: DrumsConverterArgs) {
+    super(args);
```

Comment:
We need to set the depth to 2 ** pitchClasses.length.
Since it is read-only, we may just want to not making this extend DrumsConverter and give it its own constructor.

Reply:
Actually, I think you can just re-define depth as a readonly property in this class to allow it to be overwritten.